### PR TITLE
YOGA-6 add bio page styling

### PR DESCRIPTION
**Pull request by angeliquerivera**

### CODE REVIEW

File: app/bio/page.tsx [hunk 1]
```diff
@@ -1,28 +1,78 @@
...
+                My goals are to help people understand that:{" "}
+                <blockquote className="indent-28">
+                  {" "}
+                  1. EVERYONE AND ANYONE can do yoga
+                </blockquote>{" "}
+                <blockquote className="indent-28">
+                  2. The MANY benefits of yoga
+                </blockquote>
+              </p>
+            </blockquote>
```
Comment (chrismejia) [inline app/bio/page.tsx:64]: You never want to nest `blockquote` elements: [MDN | Blockquote](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/blockquote)

You also don't need to manually number ordered lists, just use `ol` tags in combination with `li`.

```suggestion
            <blockquote className={montserrat.className}>
              <p>My goals are to help people understand that:</p>
                 <ol>
                   <li>EVERYONE AND ANYONE can do yoga</li>
                   <li>The MANY benefits of yoga</li>
                 </ol>
            </blockquote>
```

File: app/bio/page.tsx [hunk 2]
```diff
@@ -1,28 +1,78 @@
...
     <main>
-      <h2>Bio</h2>
-      <div style={{ display: "flex", flexDirection: "row", gap: "16px" }}>
+      <div
+        style={{
+          display: "flex",
+          flexDirection: "row",
+          justifyItems: "center",
+          gap: "16px",
+        }}
```
Comment (chrismejia) [inline app/bio/page.tsx:23]: Inline styles like these are great for quick development and debugging, but should be made into a formal CSS class name that you can reference. Feel free to create a new sheet that's for your styles only, and create them them there using Tailwind's built-ins. Then import it alongside the `styles.css` file and you can access them in your React pages.

File: app/styles.css [hunk 3]
```diff
@@ -0,0 +1,594 @@
...
+
+.font-bold {
+  font-weight: 700;
+}
+
+.underline {
+  text-decoration-line: underline;
+}
+
+.layout {
```
Comment (chrismejia) [inline app/styles.css:585]: You may want to be more specific in your class naming: `layout` for what element? page? sub-element? Look into BEM, SMACSS, and other CSS naming/ordering conventions.

Comment (chrismejia) [reply app/styles.css:585]: Ignore this if this is copypasta from Tailwind and is just their boilerplate.

**Review by chrismejia — CHANGES_REQUESTED** [review]

File: app/bio/page.tsx (hunk 1 above)
Comment (angeliquerivera) [reply app/bio/page.tsx]: Edited and committed

File: app/bio/page.tsx (hunk 2 above)
Comment (angeliquerivera) [reply app/bio/page.tsx]: edited and committed changes

**Review by chrismejia — APPROVED** [review]
Looking forward to layouts and designs 👍🏼

**Pull request merged**